### ISSUE TITLE
Windows paths no correct on sidebar

### ISSUE THREAD

**Issue by icetee** [atom/markdown-preview#422]
### Prerequisites

* [X] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode)?
* [X] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
* [X] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
* [X] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
* [X] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
* [X] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.

### Description
No show and not copy correct path on sidebar after load markdown-preview. **Before** running safe mod, i got encodeURI version, and no loaded markdown-preview. _(After running eliminated)_

### Steps to Reproduce

1. Open Settings view
2. Click Packages
3. Select on Settings button
4. Click LICENCE or CHANGELOG

Or run:
```
var _path = 'C:\\Users\\_USERNAME_\\AppData\\Local\\atom\\app-1.10.0\\resources\\app.asar\\node_modules\\settings-view\\LICENSE.md';

atom.workspace.open("markdown-preview://" + encodeURI(_path));
```

**Expected behavior:** [What you expected to happen]
C:\Users\icetee\AppData\Local\atom\app-1.10.0\resources\app.asar\node_modules\language-css\LICENSE.md  
**Actual behavior:** [What actually happened]
\Users\icetee\AppData\Local\atom\app-1.10.0\resources\app.asar\node_modules\language-css\LICENSE.md

### Versions

Windows 10 Pro x64

Atom    : 1.10.0
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0

apm  1.12.5
npm  3.10.5
node 4.4.5
python 2.7.11
git 2.8.2.windows.1
visual studio

### NOTE
More repositories can be affected. I'm not sure that there will actually here be a mistake.

**Comment by 50Wliu** [comment]
I cannot reproduce this on Atom 1.12.0-dev-d34e092, Windows 10 x64.

**Comment by Ben3eeE** [comment]
Can reproduce on 1.11-beta5 but not on 1.12.0-dev-24c8343 so this issue will be fixed in Atom 1.12.
